### PR TITLE
Update eaccess from 1.13.1 to 1.13.2

### PR DESCRIPTION
**Pull request by Macintron**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/eaccess.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'eaccess' do
-  version '1.13.1'
-  sha256 'c05daad636bef0c5ccc58ee75138bfc2d9da76c25af7a16664fbc9f756209546'
+  version '1.13.2'
```
Comment (miccal) [inline Casks/eaccess.rb:2]: Please change this to:
```
version '1.13.2,97676'
```

File: Casks/eaccess.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha256 'edb3eb022f2147e12d722f234ef12c7ec5c3f937004f3dd6e0165883fc5e7de1'
 
-  url 'https://glutz.com/service/downloads/?dwnldid=97482'
+  url 'https://glutz.com/service/downloads/?dwnldid=97676'
```
Comment (miccal) [inline Casks/eaccess.rb:5]: Please change this to:
```
url "https://glutz.com/service/downloads/?dwnldid=#{version.after_comma}"
```

File: Casks/eaccess.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   appcast 'https://macupdater.net/cgi-bin/check_urls/check_url_filename.cgi?url=https://glutz.com/service/downloads/?dwnldid=97482',
           must_contain: version.dots_to_underscores
```
Comment (miccal) [inline Casks/eaccess.rb:7]: Please change this to:
```
must_contain: version.before_comma.dots_to_underscores
```

**Comment by core-code** [comment]
while we are at it, the appcast doesn't seem to function to discover new version since the download URL token number changes. 

maybe we have to use the HTML page instead

https://glutz.com/service/downloads/soft-und-firmware/

**Pull request merged**